lib/API.jsx

```diff
@@ -614,6 +614,43 @@ export default function API(network, args) {
             return performPOSTRequest(commandName, parameters);
         },
 
+        /**
+         * Performs API command CreateLogin
+         * This method gets called to add a new login method to the Logins table
+         *
+         * @param {Object} parameters
+         * @param {String} parameters.partnerName
+         * @param {String} parameters.partnerPassword
+         * @param {String} parameters.partnerUserID
+         * @param {String} parameters.partnerUserSecret
+         *
+         * @returns {APIDeferred}
+         */
+        Create_Login(parameters) {
```

Comment:
```suggestion
        createLogin(parameters) {
```
I don't think there's a specific reason we do the capitalization in the other cases above unless I'm missing something, but this is a bit more consistent with other `verbNoun` api calls